index.html

```diff
@@ -0,0 +1,147 @@
+                
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+           
+
+        </section>
+        <section id="about-me-container">
+
+            <div class="about-me-block">
+                <h3 class="works-title yelow abt-me">About me</h3>
+                <p class="font-montesori  white">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="see-project-btn">Get My Resume</button>
+            </div>
+    
+            <ul  class="categories-container font-montesori white">
```

Comment:
- [ ] Kindly fix the alignment if the list of technologies to match the figma template. You are expected to stick to the design as much as possible. The technologies are center aligned on the template.
### Your design
![IMG_20221202_181815](https://user-images.githubusercontent.com/94551484/205349143-214535d8-c71f-4e1e-a99e-8a35a8876715.jpg)
### Figma template 
![IMG_20221202_181739](https://user-images.githubusercontent.com/94551484/205349156-bdf27c07-af49-4b97-aec4-3d24549a9815.jpg)